src/renderer/styles/core.scss

```diff
@@ -1,25 +1,24 @@
+@use "font";
```

Comment:
noice we using dart sass instead of node sass

Reply:
yup but its gonna go away later haha